src/turbo_flask/turbo.py

```diff
@@ -4,9 +4,9 @@
 from markupsafe import Markup
 
 
-_CDN = 'https://cdn.skypack.dev'
-_PKG = '@hotwired/turbo'
-_VER = 'v7.0.1-Ugtf0P5kipzmpINBLqbM'
+_CDN = "https://cdn.skypack.dev"
+_PKG = "@hotwired/turbo"
+_VER = "v7.1.0-RBjb2wnkmosSQVoP27jT"
```

Comment:
Could you please keep the quote style, so that it is consistent with the rest of the file? Thanks!

Reply:
Right, I'm sorry :D It's back as single quote